<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextAdventureGame</class>
 <widget class="QWidget" name="TextAdventureGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextAdventureGame</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>701</width>
     <height>429</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="consoleOutput">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="commandHistory">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter Command: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commandLine"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
